src/components/common/TmBalance.vue

```diff
@@ -730,6 +761,10 @@ select option {
   padding-left: 1rem;
 }
 
+.icon-button-container {
+  margin-right: 5px;
```

Comment:
use `rem` not pixels

Reply:
It is actually the same in the end of the day no?

![image](https://user-images.githubusercontent.com/40721795/81066890-1b4aeb00-8ede-11ea-8764-fc12cf16407d.png)
(our variables.css)

I changed it anyways

Reply:
no. `5px` is `5px` ... `var(--px)` is `0.0625rem`.